src/odemis/gui/cont/acquisition/sparc_acq.py

```diff
@@ -229,19 +235,33 @@ def on_interlock_change(self, value):
             # if the value did not change in respect to the previous value
             return
 
-        logging.warning(f"Interlock status changed from {not value} -> "
-                        f"{value}")
+        logging.warning(f"Interlock status changed from {not value} -> {value}")
 
+        ebeam_str = ""  # to be added to the message if the e-beam blanker is forced
         if value:
+            # The laser is connected electronically to the interlock, so nothing to do in software.
+            # For the e-beam (to protect the user from X-rays), we force the blanker.
+            if self._ebeam_blanker:
+                self._pre_interlock_blanker = self._ebeam_blanker.value
+                self._ebeam_blanker.value = True
+                ebeam_str = "and e-beam "
+
             popup.show_message(wx.GetApp().main_frame,
                                title="Laser safety",
-                               message=f"Laser was suspended automatically due to interlock trigger.",
+                               message=f"Laser {ebeam_str}were suspended automatically due to interlock trigger.",
```

Comment:
A minor comment, but if there is no `ebeam_str` the sentence should be `Laser was suspended` and not `Laser were suspended`. It's fine by me if you leave it like this, though.

Reply:
I've stopped trying to create fancy sentences. It's not two different sentences, in a `if`.